### ISSUE TITLE
Remove the Speaker Deck shortcode from Hugo

### ISSUE THREAD

**Issue by onedrawingperday** [gohugoio/hugo#4830]
Related: #4764 

tldr: **Nobody is using Speaker Deck because the default internal shortcode does not create a presentation embed for all..**

In my investigation of the Speaker Deck shortcode I discovered that the original author of this shortcode has included the following:
```
<script async class='speakerdeck-embed' data-id='{{ index .Params 0 }}' data-ratio='1.33333333333333' src='//speakerdeck.com/assets/embed.js'></script>
```

- The data-ratio value is crucial for the script to work because it calculates the height of a presentation. 

- This value can be different in presentations because slides can have different dimensions.

- There is no way to fetch the correct value with Hugo unless the user enters it in the shortcode input.

- The data-ratio value 1.33333333333333 that is present in the default Hugo shortcode refers **only** to presentations with a 4:3 aspect ratio like the presentation by Steve Francia that is displayed in the Hugo Docs.

For example here is the script from another presentation
```
<script async class="speakerdeck-embed" data-id="510aed10c64e463c907215787e643f7e" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
```
Note that in the above the data-ratio is 1.77777777777778 because this presentation has an aspect ratio of 16:9.

**The current shortcode is broken**

To reproduce on your own please enter the `data-id` of the second presentation in one of your Hugo projects: `{{< speakerdeck 510aed10c64e463c907215787e643f7e >}}`

Absolutely nothing will be rendered.

I vote for the removal of this shortcode from Hugo. 
If anyone was really using it there would have been open issues about it by now.

CC
@bep @digitalcraftsman @kaushalmodi @regisphilibert @budparr @moorereason @it-gro @rdwatters and everyone else

**Comment by onedrawingperday** [comment]
This has been reported in the Forum some time ago.

https://discourse.gohugo.io/t/speakerdeck-shortcode-leaves-blank-space/3191/

The user asked help about the shortcode's white space.

But on my end I get nothing rendered for presentations with 16:9 aspect ratio.

Also note that Speaker Deck is no longer is associated with GitHub and things have changed a bit with  this service since yesterday.

**Comment by kaushalmodi** [comment]
+1

**Comment by bep** [comment]
OK, @onedrawingperday  -- can you remove it and also remove the relevant documentation from this repo? 

Also, to make sure we mention it in bold in release notes: Put a line in here: #4831

**Comment by onedrawingperday** [comment]
Ok @bep I can remove this. I will send the Pull Request tomorrow.
